src/ui/mod.rs

```diff
@@ -75,7 +76,36 @@ impl<R: ResourcesProvider> Ui<R> {
 }
 
 pub fn print_error(err: &Error) {
-    println!("{:?}", err);
+    for (i, error) in err.chain().enumerate() {
+        let prefix = if i == 0 {
+            "".to_owned()
+        } else {
+            "\t".to_owned()
```

Comment:
Printing tabs in the terminal might be unpredictable, though copying can be nicer. On my terminal this is 8 spaces wide by default, which seems excessive.